src/main/java/com/example/GymProject/controller/TraineeController.java

```diff
@@ -0,0 +1,130 @@
+package com.example.GymProject.controller;
+
+import com.example.GymProject.dto.TraineeDto;
+import com.example.GymProject.dto.UserDto;
+import com.example.GymProject.mapper.EntityMapper;
+import com.example.GymProject.dto.request.DeleteRequest;
+import com.example.GymProject.dto.request.UserPassRequest;
+import com.example.GymProject.dto.request.traineerRquest.GetTraineeTrainingsRequest;
+import com.example.GymProject.dto.request.traineerRquest.TraineeRegistrationRequest;
+import com.example.GymProject.dto.request.traineerRquest.UpdateTraineeProfileRequest;
+import com.example.GymProject.dto.request.traineerRquest.UpdateTraineeTrainersRequest;
+import com.example.GymProject.dto.response.GetTrainingResponse;
+import com.example.GymProject.dto.response.UserPassResponse;
+import com.example.GymProject.dto.response.traineeResponse.GetTraineeProfileResponse;
+import com.example.GymProject.dto.response.traineeResponse.TrainerForTraineeResponse;
+import com.example.GymProject.dto.response.traineeResponse.UpdateTraineeProfileResponse;
+import com.example.GymProject.service.TraineeService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/trainees")
+public class TraineeController {
+
+    @Autowired
+    private TraineeService traineeService;
+    @Autowired
+    private EntityMapper entityMapper;
+    private static final Logger logger = LoggerFactory.getLogger(TraineeController.class);
+
+
+    @PostMapping("/register")
+    public ResponseEntity<UserPassResponse> createTrainee(@Valid @RequestBody TraineeRegistrationRequest registrationRequest) {
+        logger.info("Received request to create trainee: {}", registrationRequest);
+        UserDto userDto = new UserDto(registrationRequest.getFirstName(), registrationRequest.getLastName());
+        TraineeDto traineeDto = new TraineeDto(null, registrationRequest.getDateOfBirth(), registrationRequest.getAddress(), userDto, null);
+        UserPassResponse response = traineeService.createTrainee(traineeDto);
+        logger.info("Trainee created successfully: {}", response);
```

Comment:
Here you're logging a password within the `response` object. Here you can log response.getId();